src/analog.jl

```diff
@@ -71,7 +71,7 @@ function read(t::AITask, precision::DataType, num_samples_per_chan::Integer = -1
     num_channels = getproperties(t)["NumChans"][1]
     num_samples_per_chan_read = Int32[0]
     buffer_size = num_samples_per_chan==-1 ? 1024 : num_samples_per_chan
-    data = Array{precision}(buffer_size*num_channels)
+    data = Array{precision}(undef, buffer_size*num_channels)
```

Comment:
commits 63417e, 59fc2734, 6e24dba, d05c5f13, 91c5839fd3, 9ed588b5, and d8baa7665 all appear to address deprecation warnings.  if so, so as not to clutter the git log, could you please rebase this into a single commit with the message "fix dep warns" or similar?